external/@wordpress/elements/index.js

```diff
@@ -10,7 +16,7 @@
  * @return {wp.Element}                  Element
  */
 export function createElement( type, props, ...children ) {
```

Comment:
Another alternative is simply use aliases in webpack if in the future is decided to use a different library than React.